packages/devtools_app/lib/src/inspector/inspector_screen.dart

```diff
@@ -420,23 +448,21 @@ class ErrorNavigator extends StatelessWidget {
   }
 
   void _previousError() {
-    var newIndex = errorCount == 0
-        ? null
-        : selectedErrorIndex == null
-            ? errorCount - 1
-            : selectedErrorIndex - 1;
+    final errorIndex = widget.selectedErrorIndex.value;
+    var newIndex =
+        errorIndex == null ? widget.errors.length - 1 : errorIndex - 1;
     while (newIndex < 0) {
-      newIndex += errorCount;
+      newIndex += widget.errors.length;
     }
-    onSelectedErrorIndexChanged(newIndex);
+
+    setState(() => widget.selectedErrorIndex.value = newIndex);
   }
 
   void _nextError() {
-    final newIndex = errorCount == 0
-        ? null
-        : selectedErrorIndex == null
-            ? 0
-            : (selectedErrorIndex + 1) % errorCount;
-    onSelectedErrorIndexChanged(newIndex);
+    final errorIndex = widget.selectedErrorIndex.value;
+    final newIndex =
+        errorIndex == null ? 0 : (errorIndex + 1) % widget.errors.length;
+
+    setState(() => widget.selectedErrorIndex.value = newIndex);
```

Comment:
instead of calling set state here for previous and next, call a method on the controller `selectError(int index)` (referenced in another comment earlier) that sets the value of the notifier. Then this widget will rebuild automatically because it will be wrapped in a ValueListenableBuilder

Reply:
Done in eee1fd77!